doc/crud.md

```diff
@@ -0,0 +1,106 @@
+# Access a document
+
+#### Request
+```http
+GET /data/:type/:id
```

Comment:
Don't forget that type is expected to be namespaces, like `github.com/cozy/cozy-types/events`